### DISCUSSION TITLE
mutate function is not working

### DISCUSSION THREAD

**Discussion by deepikazuci** [TanStack/query#3190] [Q&A]
I am using 2 post functions in one of the components. I used `useMutation` Hook for both. While one of the `mutate` function is working absolutely fine. I couldn't find why the other `mutate` function is not working. 

![post_ReactQuery](https://user-images.githubusercontent.com/95339477/149307190-b614d976-4403-49d5-9bf3-d32b5a4ac8c1.PNG)
![post_RegularFunction](https://user-images.githubusercontent.com/95339477/149307280-de619699-da99-4db6-8515-f280d32991bd.PNG)

I somehow managed to solve this issue by using regular Fetch function in combination with `refetch` from useQuery. 
May I know is there any configuration to make this work using `mutate` function from `useMutation`???

**Comment by TkDodo** [comment]
not sure how this screenshots should be of any help? please post your code that calls useMutation - best in a reproducible codesandbox example, thank you.
